app/frontend/entrypoints/application.js

```diff
@@ -36,7 +36,13 @@ import "/js/TestDestination";
 import "/js/Tooltips";
 import "/js/Toasts";
 import "/js/CollapseScroll";
-import "/js/formModals";
+
```

Comment:
is the extra space deliberate?

Reply:
Yeah I did it to group those modals together, it's probably not necessary though :)